File: Initial Prototype/Client/MyApplicationProject/MyApplication/src/main/java/com/example/myapplication/dtos/JourneyUserDTO.java
package com.example.myapplication.dtos;

/**
 * DTO object used when a user withdraws from a journey or by the driver when they decide to cancel one of their journeys.
 */
public class JourneyUserDTO {
    private int JourneyId;
    private int UserId;

    public JourneyUserDTO(int journeyId, int userId) {
        this.JourneyId = journeyId;
        this.UserId = userId;
    }
}

File: Project/WebService/Services/DTOs/JourneyUserDTO.cs
// --------------------------------------------------------------------------------------------------------------------
// <copyright file="JourneyUserDTO.cs" company="">
//   
// </copyright>
// <summary>
//   Defines the JourneyUserDTO type.
// </summary>
// --------------------------------------------------------------------------------------------------------------------

namespace Services.DTOs
{
    using System.Runtime.Serialization;

    /// <summary>
    /// The journey user dto.
    /// </summary>
    [DataContract]
    public class JourneyUserDTO
    {
        /// <summary>
        /// Gets or sets the user id.
        /// </summary>
        [DataMember]
        public int UserId { get; set; }

        /// <summary>
        /// Gets or sets the journey id.
        /// </summary>
        [DataMember]
        public int JourneyId { get; set; }
    }
}